test/2.0/csv.spec.ts

```diff
@@ -722,11 +956,313 @@ test("validateRow wide conditionals", (t) => {
+      "consult the appendix",
+    "estimated_amount | Payer One | Basic Plan": "245",
+    "standard_charge | Payer One | Basic Plan | methodology": "fee schedule",
+  }
+  const algorithmWithMethodologyErrors = validateRow(
+    algorithmWithMethodologyRow,
+    11,
+    columns,
+    true
+  )
+  t.is(algorithmWithMethodologyErrors.length, 0)
+  const algorithmNoMethodologyRow = {
+    ...basicRow,
+    "standard_charge | Payer One | Basic Plan | negotiated_algorithm":
+      "consult the appendix",
+    "estimated_amount | Payer One | Basic Plan": "245",
+  }
+  const algorithmNoMethodologyErrors = validateRow(
+    algorithmNoMethodologyRow,
+    12,
+    columns,
+    true
+  )
+  t.is(algorithmNoMethodologyErrors.length, 1)
+  t.assert(
+    algorithmNoMethodologyErrors[0].message.includes(
+      '"standard_charge | Payer One | Basic Plan | methodology" is required when a payer specific negotiated charge is encoded as a dollar amount, percentage, or algorithm'
+    )
+  )
+  const algorithmWrongMethodologyRow = {
+    ...basicRow,
+    "standard_charge | Payer One | Basic Plan | negotiated_algorithm":
+      "consult the appendix",
+    "estimated_amount | Payer One | Basic Plan": "245",
+    "standard_charge | Payer Two | Special Plan | methodology": "fee schedule",
+  }
+  const algorithmWrongMethodologyErrors = validateRow(
+    algorithmWrongMethodologyRow,
+    12,
+    columns,
+    true
+  )
+  t.is(algorithmWrongMethodologyErrors.length, 1)
+  t.assert(
+    algorithmWrongMethodologyErrors[0].message.includes(
+      '"standard_charge | Payer One | Basic Plan | methodology" is required when a payer specific negotiated charge is encoded as a dollar amount, percentage, or algorithm'
+    )
+  )
+  const algorithmInvalidMethodologyRow = {
+    ...basicRow,
+    "standard_charge | Payer One | Basic Plan | negotiated_algorithm":
+      "consult the appendix",
+    "estimated_amount | Payer One | Basic Plan": "245",
+    "standard_charge | Payer One | Basic Plan | methodology":
+      "special methodology",
+  }
+  const algorithmInvalidMethodologyErrors = validateRow(
+    algorithmInvalidMethodologyRow,
+    13,
+    columns,
+    true
+  )
+  t.is(algorithmInvalidMethodologyErrors.length, 1)
+  t.assert(
+    algorithmInvalidMethodologyErrors[0].message.includes(
+      '"standard_charge | Payer One | Basic Plan | methodology" value "special methodology" is not one of the allowed values'
+    )
+  )
+
+  // If the "standard charge methodology" encoded value is "other", there must be a corresponding explanation found
+  // in the "additional notes" for the associated payer-specific negotiated charge.
+  const otherWithNotesRow = {
+    ...basicRow,
+    "standard_charge | Payer One | Basic Plan | negotiated_percentage": "80",
+    "estimated_amount | Payer One | Basic Plan": "150",
+    "standard_charge | Payer One | Basic Plan | methodology": "other",
+    "additional_payer_notes | Payer One | Basic Plan":
```

Comment:
This wouldn't satisfy the "additional notes" being populated for the specific plan's "other" in methodology?